components/ModalCard/index.jsx

```diff
@@ -0,0 +1,59 @@
+import React, {PropTypes} from 'react';
+import classnames from 'classnames';
+import {
+  modifierClassList, defaultReactProps, defaultReactPropsValues
+} from '../utils';
+
+const ModalCardFooter = ({children}) => {
+  return (
+    <footer className="modal-card-foot">
+      {children}
+    </footer>
+  );
+};
+
+ModalCardFooter.propTypes = {
+  children: PropTypes.arrayOf(PropTypes.element)
+};
+
+const ModalCard = ({title, footer, className, children, ...props}) => {
+  let {classList, ...finalProps} = modifierClassList(props);
+  classList = classnames('modal', className, classList);
+
+  const header = (
+    <header className="modal-card-head">
+      <p className="modal-card-title">{title}</p>
+      <button className="delete"></button>
+    </header>
+  );
+
+  return (
+    <div className={classList} style={props.visible ? {'display': 'block'} : {}}>
+      <div className="modal-background"></div>
+      <div className="modal-card">
+        {header}
```

Comment:
Not sure I see the value in placing header in a variable, may as well just put the code here.